src/main/scala/mimir/util/TimeUtils.scala

```diff
@@ -25,4 +28,23 @@ object TimeUtils {
     }
   }
 
+  def getDaysBetween(start: PrimitiveValue, end: PrimitiveValue): Long = {
+    val startDT = start.asDateTime
+    val endDT = end.asDateTime
+    if(startDT.isBefore(endDT))  
+      Days.daysBetween(startDT, endDT).getDays() 
+    else
+      Days.daysBetween(startDT, endDT).getDays() 
+  }
+
+  def getSecondsBetween(start: PrimitiveValue, end: PrimitiveValue): Long = {
+    val startDT = start.asDateTime
+    val endDT = end.asDateTime
+    if(startDT.isBefore(endDT))
+      Seconds.secondsBetween(startDT, endDT).getSeconds()
+    else
+      Seconds.secondsBetween(startDT, endDT).getSeconds()
+  }
+
+  
```

Comment:
Maybe move the old TimeUtils functions into a new file/object called `Timer`.  These functions are thematically quite different.

Reply:
By old TimeUtils Function you meant mark() and monitor() ?